src/components/ColumnItem.tsx

```diff
@@ -0,0 +1,113 @@
+import { useState } from 'react'
+import { useFirestore } from 'react-redux-firebase'
+import { useSelector } from 'react-redux'
+import {
+  Paper,
+  IconButton,
+  TextField,
+  Typography,
+  Divider,
+} from '@mui/material'
+import CheckIcon from '@mui/icons-material/Check'
+import DeleteIcon from '@mui/icons-material/Delete'
+import EditIcon from '@mui/icons-material/Edit'
+import { CardList } from './CardList'
+import { CardForm } from './CardForm'
+import { Card, WithID, RootState } from '../store/store'
+
+export const ColumnItem = (props: {
+  cards: WithID<Card>[]
+  title: string
+  id: string
+  key: string
+}) => {
```

Comment:
```suggestion
interface Props {
  cards: WithID<Card>[]
  id: string
  title: string
}

export const ColumnItem = ({ cards, id, title }: Props) => {
```

Reply:
You also don't need `key` here. You're not using it anywhere, and it has the same value as `id`.